ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/DataHubPlugin.groovy

```diff
@@ -115,14 +115,14 @@ class DataHubPlugin implements Plugin<Project> {
                 "This will first download the artifacts (entity models, flows, step definitions, and steps) , then delete all user files in the project directory"+
                 " and the contents of the downloaded zip will then be extracted into the project directory.")
 
-        String flowMigrationGroup = "Data Hub Flow Migration"
-        project.task("hubDeleteInstalledLegacyMappings", group: flowMigrationGroup, type: DeleteInstalledLegacyMappingsTask,
+        String hubMigrationGroup = "Data Hub Migration"
+        project.task("hubDeleteInstalledLegacyMappings", group: hubMigrationGroup, type: DeleteInstalledLegacyMappingsTask,
             description: "Delete installed legacy mappings, which are mappings that have not been converted into the format required by Hub Central"
         ).mustRunAfter("hubDeployUserArtifacts")
-        project.task("hubMigrateProjectFlows", group: flowMigrationGroup, type: MigrateProjectFlowsTask,
+        project.task("hubMigrateForHubCentral", group: hubMigrationGroup, type: MigrateProjectForHubCentralTask,
             description: "Migrate flows and mappings in the local project that were created before version 5.3.0 into the new format required for usage within Hub Central"
```

Comment:
For the description, change to "Migrate flows, mappings, and entity models in the local project..."